include/libtorrent/netlink.hpp

```diff
@@ -106,6 +106,9 @@ namespace libtorrent {
 			return sizeof(m_sockaddr);
 		}
 
+		// commented the comparison operators for now, until the
+		// same operators are implemented for sockaddr_nl
+		/*
```

Comment:
I would imagine these operators would need to be implemented as ``std::tie(l.m_sockaddr. X, ...) < std::tie(r.m_sockaddr. X, ...)`` or something like that. assuming we need them at all